apps/website/src/server/db/short-links.ts

```diff
@@ -0,0 +1,52 @@
+import { z } from "zod";
+import { TRPCError } from "@trpc/server";
+import { prisma } from "@/server/db/client";
+import { convertToSlug, SLUG_REGEX } from "@/utils/slugs";
+
+export const shortLinkSchema = z.object({
+  label: z.string(),
+  slug: z.string().regex(SLUG_REGEX).optional(),
```

Comment:
With the forms not requiring a label or slug was fine because we have a fallback to the id there. For the short links I think we should check there is a slug (or a label to infer the slug from). Not sure if thats easily doable with zod, so maybe just always require a non empty label.